readthedocs/templates/doc_builder/conf.py.tmpl

```diff
@@ -148,5 +148,6 @@ else:
 # Add custom RTD extension
 if 'extensions' in globals():
     extensions.append("readthedocs_ext.readthedocs")
+    extensions.append("docs_italia_theme")
```

Comment:
We need it because this theme is not only a collection of static templates and assets but include also many features added as extensions in its `__init__.py`. If we don't add the theme as an extension they don't work.

Reply:
ok then